### DISCUSSION TITLE
Table 的「網狀的」效果在 firefox browser 上不明顯

### DISCUSSION THREAD

**Discussion by Alpaca0x0** [TeaMeow/TocasUI#841] [Ideas]
你好，想回報一個小小的問題！ ◝(　ﾟ∀ ﾟ )◟

使用 Table 時，如果同時使用 Class `.is-celled` 及 `.is-striped`
其網格效果在 **Firefox browser** 上有**顯示不明顯**的狀況：

### Firefox ٩(◦`꒳´◦)۶

![image](https://user-images.githubusercontent.com/58300460/183067720-8bbce2ca-8b3a-4aac-a578-13c1b1ff1b0b.png)

### Chrome ⸜(* ॑꒳ ॑* )⸝

![image](https://user-images.githubusercontent.com/58300460/183067658-065688ce-e1be-4eba-a96f-c2ff99e22715.png)

---

OS：`Ubuntu 20.04.4 LTS`
Firefox Version：`103.0 (64-bit)`

**Comment by YamiOdymel** [comment]
Firefox 11 年前的 Bug 到現在還沒有修好。主要是 `<tr>` 的背景顏色會蓋過 `<td>` 的框線

官方 Bug 追蹤：
[Background of cell paints over border when borders are collapsed and tbody, row, or cell is relatively positioned](https://bugzilla.mozilla.org/show_bug.cgi?id=688556)

討論：
[Borders not shown in Firefox with border-collapse on table, position: relative on tbody, or background-color on cell](https://stackoverflow.com/questions/7517127/borders-not-shown-in-firefox-with-border-collapse-on-table-position-relative-o)

再來看看怎麼處理

**Reply by Alpaca0x0** [reply]
原來如此，看這樣子短時間內應該不會有變動
總之非常感謝！⸜(* ॑꒳ ॑* )⸝
